api/src/restful/use_case.py

```diff
@@ -1,38 +1,62 @@
 import json
 from dmss_api.exceptions import ServiceException, UnauthorizedException
-
+from starlette.responses import JSONResponse
 from repository.repository_exceptions import (
     ApplicationNotLoadedException,
     FileNotFoundException,
     EntityNotFoundException,
     JobNotFoundException,
 )
-from restful import response_object as res
+
 import traceback
+from starlette import status
+
+
+def create_error_response(
+    error: Exception,
+    status: int,
+) -> JSONResponse:
+    types = {
+        400: "PARAMETERS_ERROR",
+        401: "UNAUTHORIZED",
+        403: "FORBIDDEN",
+        404: "RESOURCE_ERROR",
+        422: "UNPROCESSABLE_ENTITY",
+        500: "SYSTEM_ERROR",
+    }
```

Comment:
Not important, but perhaps we could reuse the `STATUS_CODES` from enums.py somehow?

Reply:
good point, it actually looks like this types dict is not needed. we can use the status dict from starlette, which gives more consistency.

This means we can delete the STATUS_CODES in enums.py also